src/components/ResourceCard/AddResource/style.scss

```diff
@@ -1,5 +1,19 @@
 .resource-activity {
   // padding-left: 25px;
+   #laravel-h5p-form{
+     .methods .control-label{
+      font-weight: bold;
+     }
+
+     .methods .radio-inline{
+      padding-right: 10px;
+      cursor: pointer;
+     }
+     
+     .methods .radio-inline input{
+      margin-right: 2px;
+     }
+  }
```

Comment:
It seems like this part is not correct indented. It should throw eslint error.

Reply:
eslint didn't through error on it. seems to be ok then

Reply:
It really strange that Eslint errors are not throwing on your local
I will add some script and please run that command before make PR in future